model/src/main/kotlin/Identifier.kt

```diff
@@ -102,7 +102,7 @@ data class Identifier(
             val lowerName = name.lowercase()
             val vendorNamespace = when (type) {
                 "NPM" -> "@$lowerName"
-                "Gradle", "Maven", "SBT" -> "(com|io|net|org)\\.$lowerName(\\..+)?"
+                "Gradle", "Maven", "SBT" -> "(com|io|net|org)?\\.?$lowerName(\\..+)?"
```

Comment:
This would also allow a namespace with a leading dot. I guess that's not allowed in Maven. So you probably need to write this as `((com|io|net|org)\\.)?$lowerName(\\..+)?`

Reply:
Also the commit message is a bit misleading, IMO. It's not that the `Identifier` class did not allow namespaces in non-TLD form, it's just that the `isFromOrg` did not work as expected in that case.